modules/products/bitbucket/nfs/main.tf

```diff
@@ -1,13 +1,62 @@
+resource "aws_ebs_volume" "shared_home" {
```

Comment:
Are we going to use nfs for share home of all products? in this case I think better we move nfs out of bitbucket module put in in `modules/AWS`

Reply:
We will do it as part of other story.